contributing-docs/03_contributors_quick_start.rst

```diff
@@ -500,7 +511,7 @@ on macOS, install via
 
   brew install libxml2
 
-2. Installing pre-commit (if you have not done it yet):
+2. Installing pre-commit:
```

Comment:
I think better remove this section of install pre-commit using uv and keep the upper section, to have a flow of how to install pre-commit properly

Reply:
Yeah we should default to using `uv tool install` instead of `pipx` - it's an omission that we still have it here.

Reply:
If I understood you correctly, done.